ONSdigital/eq-author-app
# BLOCKED - EAR 1810 Remove second radio option

### What is the context of this PR?

> Remove unused radio buttons when one is selected as dynamic answer

### How to review

> Add a question with a checkbox answer with a number of options
> Add a question with radio answers
> Have some with labels and some without
> Select one radio as a Dynamic Answer
> Check that the unused radio buttons are removed

> Add to the list below as appropriate, including screenshots when necessary

- Import the **Capability examples** questionnaire from pre-prod Author into your local environment and:
  - ensure it can be opened in Author;
  - then, ensure it can be viewed in Runner by pressing the **view survey** button
- Does this require a migration? We need one if existing JSON schema properties change

### What to do after everything is green

1. - [ ] Bring this branch up-to-date with Origin/Master
2. - [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. - [ ] Click the **Merge** button on this pull request
4. - [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. - [ ] Move the Jira ticket for this task into the next stage of the process
